builtin/providers/nsone/resource_record.go

```diff
@@ -824,12 +339,20 @@ func RecordCreate(d *schema.ResourceData, meta interface{}) error {
 
 // RecordRead reads the DNS record from ns1
 func RecordRead(d *schema.ResourceData, meta interface{}) error {
-	client := meta.(*nsone.Client)
+	//client := meta.(*nsone.Client)
```

Comment:
Thank you @cakoose, i missed that one at first.

Just posted a [PR](https://github.com/hashicorp/terraform/pull/11300) against the terraform ns1-provider branch. Probably a bit more work to do, but its almost there :)